README.md

```diff
@@ -6,6 +6,15 @@ The sample application consists of two services:
 * A stateless service `frontend`
 * A stateful entity-based service `shopping-cart`
 
+## IDE setup
+
+Eclipse, IntelliJ and VS Code are able to be used to maintain this project. With Eclipse and VS Code, it is important that the `mvn generate-sources` goal is run
```

Comment:
Suggest simplified text:

You can add the sample project to an IDE such as Eclipse, IntelliJ, or VS Code (?unless those three are the only ones we want to suggest, in which case leave out the "such as"?). Various `.proto` editing plugins are available. For xeample Eclipse offers the `protobuf-dt` plugin. When using a `protobuf` editor, you might need to specify additional folders for importing such as `target/protoc-dependencies` or `target/....`. (?It's not clear to me what I need to do about this and how would I know which folders to create?). 

NOTE: For Eclipse and VS code, run `mvn generate-sources` after loading the project and each time you change a `.proto` file.

Reply:
Thanks for the suggestions - would you mind submitting your own PR for those? In particular, I think what I had in terms of those sample import folders makes sense.

To answer your questions:

> unless those three are the only ones we want to suggest, in which case leave out the "such as"?
These are the three major ones, but there are more, including Netbeans.

> It's not clear to me what I need to do about this and how would I know which folders to create?
You're not creating folders - you are importing ones that have been generated by the Maven protoc integration... I'll add a clarification as to why you may need to import these.

Reply:
The clarified sentence now reads:

> When using a protobuf editor you may need to specify additional folders for importing so that an editor can avoid highlighting errors.